pkg/apis/serving/v1alpha1/kfservice_framework_xgboost.go

```diff
@@ -14,21 +14,27 @@ limitations under the License.
 package v1alpha1
 
 import (
+	"fmt"
+	"strings"
+
+	"github.com/kubeflow/kfserving/pkg/utils"
 	v1 "k8s.io/api/core/v1"
 )
 
-const (
-	XGBoostServingGRPCPort  = "9000"
-	XGBoostServingRestPort  = "8080"
-	XGBoostServingImageName = "animeshsingh/xgbserver"
-
-	DefaultXGBoostServingVersion = "latest"
+// TODO add image name to to configmap
+var (
+	AllowedXGBoostRuntimeVersions = []string{
+		"latest",
```

Comment:
I think this also needs to be added to ConfigMap later, how are we going to maintain the version list?

Reply:
Giving an ability to end user to override the default with supported versions they may want to use. So yes a ConfigMap with defaults which can be overridden

Reply:
Mmmm I would like to punt on configuring allowed versions.